src/parser.rs

```diff
@@ -13,6 +13,14 @@ use crate::{
 
 const EMPTY_CTYPE: &str = "/*EMPTY*/";
 
+#[derive(Debug, Default)]
+struct ParsedType {
```

Comment:
The naming here confused me a bit, could use a small comment on what it is.

Reply:
Updated